### PR TITLE
Intellij objectives

### PR DESCRIPTION
**Pull request by chrisbay**

### CODE REVIEW

**Comment by chrisbay** [comment]
[IntelliJ appendix objectives](https://trello.com/c/N50XJAzK/337-intellij-appendix-objectives)

File: unit2-java/intellij.rst [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+- Create breakpoints in source code
+- Run programs in debug mode
+- Inspect variables and parameters while in debug mode
+- Create conditional breakpoints
+
+IDE Features
+^^^^^^^^^^^^
+
+- Understand how IntelliJ uses Gradle to carry out like compiling and running code
```
Comment (jimflores5) [inline unit2-java/intellij.rst]: Remove the "like" before "compiling".

File: unit2-java/intellij.rst [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+- Understand how IntelliJ flags compiler warnings and errors
+- User refactoring tools to efficiently improve and change code
+- Use embedded tools: terminal, Git, database interface 
+- Use common keyboard shortcuts: organize imports (opt+cmd+o), import package (alt+enter)
```
Comment (jimflores5) [inline unit2-java/intellij.rst]: How about,
 - Use two common keyboard shortcuts:
    - Organize imports (`opt+cmd+o`),
    - Import package (`alt+enter`).

File: unit2-java/intellij.rst [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+- Inspect variables and parameters while in debug mode
+- Create conditional breakpoints
+
+IDE Features
+^^^^^^^^^^^^
+
+- Understand how IntelliJ uses Gradle to carry out like compiling and running code
+- Understand how IntelliJ flags compiler warnings and errors
+- User refactoring tools to efficiently improve and change code
+- Use embedded tools: terminal, Git, database interface
```
Comment (jimflores5) [inline unit2-java/intellij.rst]: How about, "Use the embedded tools to carry out terminal commands, Git commands, and database interface."

Alternatively,
- Use embedded tools to carry out:
  - Terminal actions,
  - Git commands,
  - Database interface

File: unit2-java/intellij.rst [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+
+Debugging
+^^^^^^^^^
+
+- Create breakpoints in source code
+- Run programs in debug mode
+- Inspect variables and parameters while in debug mode
+- Create conditional breakpoints
+
+IDE Features
```
Comment (jimflores5) [inline unit2-java/intellij.rst:21]: How about adding a bullet point or two about hovering over red lines/red words to pick up suggestions for fixing code errors?

**Pull request merged**